src/test/java/integrationtests/jbehave/GaragePerson.java

```diff
@@ -0,0 +1,28 @@
+package integrationtests.jbehave;
+
+import de.codecentric.jbehave.junit.monitoring.JUnitReportingRunner;
+import integrationtests.jbehave.steps.GaragePersonSteps;
+import org.jbehave.core.configuration.Configuration;
+import org.jbehave.core.configuration.MostUsefulConfiguration;
+import org.jbehave.core.io.LoadFromClasspath;
+import org.jbehave.core.junit.JUnitStory;
+import org.jbehave.core.reporters.Format;
+import org.jbehave.core.reporters.StoryReporterBuilder;
+import org.jbehave.core.steps.InjectableStepsFactory;
+import org.jbehave.core.steps.InstanceStepsFactory;
+import org.junit.runner.RunWith;
+
+@RunWith(JUnitReportingRunner.class)
+public class GaragePerson extends JUnitStory {
```

Comment:
Your right. To run multiple stories you have to extend JUnitStories. 
Example from [documentation](http://jbehave.org/reference/latest/configuration.html)
```
public class TraderStories extends JUnitStories { 
    public TraderStories() {
        // configure as TraderStory except for 
        Configuration configuration = new MostUsefulConfiguration()
                .useStoryLoader(new LoadFromURL())
    }
 
    @Override
    protected List<String> storyPaths() {
        String codeLocation = codeLocationFromClass(this.getClass()).getFile();
        return new StoryFinder().findPaths(codeLocation, asList("**/trader*.story"),
                    asList(""), "file:"+codeLocation);
    }
}
```

Reply:
Yes! I tried to do this, but i didn't have success. I don't know if i have do to more configurations. But i decided to let this way just for run.

Reply:
Leave it to another time. It's not important right now to know all configuration details.